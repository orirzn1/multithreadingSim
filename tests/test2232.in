L9
S5
O14
N5

T0
I@0x00000000
SUBI $6, $0, 0x2
ADD $0, $0, $6
ADDI $6, $3, 0x1
LOAD $3, $2, $7
SUBI $4, $7, 0x2
LOAD $7, $6, $3
SUB $6, $7, $6
ADD $5, $4, $4
HALT $0

T1
I@0x00000000
LOAD $7, $6, 0x2
LOAD $6, $0, 0x0
STORE $0, $0, $3
SUB $5, $6, $4
ADDI $1, $2, 0x4
ADDI $7, $0, 0x3
HALT $0

T2
I@0x00000000
SUB $2, $4, $7
SUBI $5, $7, 0x2
SUBI $1, $3, 0x4
SUBI $4, $5, 0x4
ADDI $3, $0, 0x4
ADDI $4, $7, 0x4
ADDI $0, $0, 0x2
ADD $4, $5, $7
SUBI $7, $1, 0x2
HALT $0

T3
I@0x00000000
LOAD $0, $7, 0x1
LOAD $7, $7, 0x3
ADDI $7, $7, 0x1
ADD $1, $0, $6
SUB $5, $7, $3
SUBI $1, $0, 0x1
STORE $1, $1, 0x4
LOAD $1, $0, $2
SUB $3, $7, $3
HALT $0

T4
I@0x00000000
LOAD $7, $1, $3
ADDI $4, $0, 0x4
ADDI $7, $3, 0x2
LOAD $7, $0, $2
ADDI $0, $0, 0x2
ADDI $7, $1, 0x1
LOAD $7, $7, $0
STORE $7, $5, $4
HALT $0

D@0x0
0x5
0x7
0x8
0x2
0x6
0x9
0x1
0x8
0x7
