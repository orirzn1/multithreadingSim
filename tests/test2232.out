
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0xFFFFFFFE	R1 = 0x0	R2 = 0x0	R3 = 0x5	R4 = 0xFFFFFFFE	R5 = 0xFFFFFFFC	R6 = 0x6	R7 = 0x7
Register file thread id 1:
	R0 = 0x0	R1 = 0x4	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x5	R6 = 0x5	R7 = 0x3
Register file thread id 2:
	R0 = 0x2	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0x4	R4 = 0xFFFFFFFE	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0xFFFFFFFA
Register file thread id 3:
	R0 = 0x5	R1 = 0x7	R2 = 0x0	R3 = 0x6	R4 = 0x0	R5 = 0x6	R6 = 0x0	R7 = 0x6
Register file thread id 4:
	R0 = 0x2	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x4	R5 = 0x0	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 6.288889

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0xFFFFFFFE	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFE	R5 = 0xFFFFFFFC	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x4	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x3
Register file thread id 2:
	R0 = 0x2	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0x4	R4 = 0xFFFFFFFE	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0xFFFFFFFA
Register file thread id 3:
	R0 = 0x0	R1 = 0xFFFFFFFF	R2 = 0x0	R3 = 0x1	R4 = 0x0	R5 = 0x1	R6 = 0x0	R7 = 0x1
Register file thread id 4:
	R0 = 0x2	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x4	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFF
Finegrained Multithreading CPI for this program 1.177778

